.gitlab-ci.example.yml

```diff
@@ -10,21 +10,38 @@
 # * Set in the schedule required variables
 #
 #     PROJECT_PATH = group/repository
-#     PACKAGE_MANAGER_SET = bundler,composer,npm_and_yarn
+#     PACKAGE_MANAGER = bundler,composer,npm_and_yarn
```

Comment:
I think the idea with `_SET` suffix is to highlight that multiple package managers can be included, it's not just one... 

I'm okay with a rename, but I don't think `PACKAGE_MANAGER` is the right choice because it implies it can be only one.

Reply:
I just basically rename the var according to the script which use this car. [The line in the script ](https://github.com/dependabot/dependabot-script/blob/main/generic-update-script.rb#L47)